lib/easel_provider.dart

```diff
@@ -35,13 +40,15 @@ class EaselProvider extends ChangeNotifier {
   final VideoPlayerHelper videoPlayerHelper;
   final AudioPlayerHelper audioPlayerHelper;
   final FileUtilsHelper fileUtilsHelper;
+  final Repository repository;
 
   EaselProvider({
     required this.localDataSource,
```

Comment:
Remove local data store and remote data store access from this provider. Use repository as one source of truth.